cwms_radar_api/src/main/java/cwms/radar/formatters/ContentType.java

```diff
@@ -46,17 +47,30 @@ public String getCharset() {
         return charset;
     }
 
+    /**
+     * For the purposes of cwms-data-api content-type equals we only care about the following
+     * fields matching:
+     * 
+     *  - the mimetype itself
+     *  - the version parameter
+     * 
+     * For us everything else is informational or used indirectly
+     */
     @Override
     public boolean equals(Object other) {
         logger.atFinest().log("Checking %s vs %s", this, other);
         if (!(other instanceof ContentType)) return false;
         ContentType o = (ContentType) other;
         if (!(mediaType.equals(o.mediaType))) return false;
 
+        /** We loop through instead of using contains key. 
+         *  Content-type parameter names are not case sensitive.
+         */
         for (Map.Entry<String, String> entry : parameters.entrySet()) {
             String key = entry.getKey();
-            if (key.equals("q")) continue; // we don't care about q for equals
-            if (!entry.getValue().equals(o.parameters.get(key))) return false;
+            if( "version".equalsIgnoreCase(key) ) {
+                return entry.getValue().equals(o.parameters.get(key));
+            }
```

Comment:
I think this change deserves some tests in ContentTypeTest to demonstrate what we expect to be equals and what isn't.

Reply:
simple test added. I'll grant the names may not be the best but for the moment I was going for change the least amount of code at one time.